src/python/gem5/components/boards/kernel_disk_workload.py

```diff
@@ -145,6 +156,7 @@ def set_kernel_disk_workload(
         disk_image: DiskImageResource,
         bootloader: Optional[BootloaderResource] = None,
         disk_device: Optional[str] = None,
+        additional_disks: List[DiskImageResource] = [],
```

Comment:
The docstring needs updated here. It should be added that the `disk_device` is assumed to contain the image with the root to be mounted. `additional_disks` needs documentation too.

Reply:
Done, I have updated the description of the disk_image and additional_disks, to address the new functionality.